### PR TITLE
Adding stats page

### PR DESCRIPTION
**Pull request by thegame8714**

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Looks nice!

BTW, a cool feature Netlify has is it can post a comment once the deploy preview is available. That'd be fun to turn on.

**Comment by gabegreenberg** [comment]
@KyleAMathews where is that feature located in Netlify? You mean "when deploy succeeds" ??
screen shot on 2017-01-12 at 16-10-05

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+}
```
Comment (vcarl) [inline .vscode/settings.json]: Shouldn't commit this file IMO. Remove it and do a rebase to remove it from history?

File: wrappers/md.js [hunk 2]
```diff
@@ -2,13 +2,43 @@ import React from 'react'
...
+    this.state = {
+      contributors:  [],
+      months: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+      isFetchingData: true
+    }
+  }
+
+  componentWillMount() {
+    if(this.state.contributors.length === 0) { 
+    let data = retrieveMonthlyHoF()
```
Comment (vcarl) [inline wrappers/md.js]: Indentation

File: wrappers/md.js [hunk 3]
```diff
@@ -2,13 +2,43 @@ import React from 'react'
...
+  constructor() {
+    super()
+    this.state = {
+      contributors:  [],
+      months: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+      isFetchingData: true
+    }
+  }
+
+  componentWillMount() {
```
Comment (huv1k) [inline wrappers/md.js]: I don't know if best idea is prefetch even if we don't enter Stats page. You will get data when you go for example to Learning page, but you don't need to visit Stats page.

**Comment by huv1k** [comment]
@thegame8714 Hey, any update? :)

**Comment by thegame8714** [comment]
Unfortunately not! I'm not able to let the Discord API work. Do you have any suggestion about it?

**Pull request closed without merging**